src/main/scala/scoverage/ScoverageKeys.scala

```diff
@@ -21,4 +26,33 @@ object ScoverageKeys {
   lazy val coverageOutputTeamCity = settingKey[Boolean]("turn on teamcity reporting")
   lazy val coverageScalacPluginVersion = settingKey[String]("version of scalac-scoverage-plugin to use")
   // format: on
+
+  def coverageMinima = Def.setting {
+    CoverageMinima(
+      total = CoverageMinimum(
+        statement = coverageMinimum.value,
+        branch = coverageMinimumBranchTotal.value
+      ),
+      perPackage = CoverageMinimum(
+        statement = coverageMinimumStmtPerPackage.value,
+        branch = coverageMinimumBranchPerPackage.value
+      ),
+      perFile = CoverageMinimum(
+        statement = coverageMinimumStmtPerFile.value,
+        branch = coverageMinimumBranchPerFile.value
+      )
+    )
+  }
+
+  case class CoverageMinimum(
+      statement: Double,
+      branch: Double
+  )
+
+  case class CoverageMinima(
+      total: CoverageMinimum,
+      perPackage: CoverageMinimum,
+      perFile: CoverageMinimum
+  )
```

Comment:
Nit, since this file is `ScoverageKeys`, can we keep it _just_ keys and possibly move this out to it's own file?

Reply:
will do